.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: 'test'
+
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  go-test:
+    name: go-test
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.21
```

Comment:
Maybe this instead? https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file

Reply:
Good idea, updated 🏓